support/editor-document.js

```diff
@@ -33,7 +32,7 @@ class EditorDocument {
       return this.topDomNode;
     } else {
       const dom = this.getDom();
-      const topDomNode = dom.window.document.querySelector('body');
+      const topDomNode = dom.window._document.querySelector('body');
```

Comment:
this doesn't seem right looking at the jsdom docs https://github.com/jsdom/jsdom

Reply:
Yeah, I found the same, but typescript complains about it, and _document seems to also work, should I ignore typescript for that line or go with _document?

Reply:
I wouldn't go for _document, it's undocumented and _ typically means it's meant for internal use only

Reply:
ook  👍